### ISSUE TITLE
Module: Save window state

### ISSUE THREAD

**Issue by Therzok** [Therzok/dsp_modding#2]
Nothing fancy, just save window state on save, so it is restored the next session.

**Comment by Simpy** [comment]
I really want to use this mod, however I seem to have a slight problem, when the mod is installed it turns everything blurry (see pics) - the only other graphical  mod I am using is UILayoutHeightFix - Screen resolution is 3840x2160 and UILayoutHeightFix is set to 2160
![with-mod](https://user-images.githubusercontent.com/447796/111930126-08a0cb00-8b1d-11eb-9dd0-9e2bd2d1b339.JPG)
![without-mod](https://user-images.githubusercontent.com/447796/111930128-0a6a8e80-8b1d-11eb-8413-7e855fd8abc1.JPG)

**Comment by Therzok** [comment]
@Simpy Oh, I think I know what's causing that. It is causing anti aliasing due to restoring on subpixel position.

Let me see if I can fix that and publish a new version!

**Comment by Therzok** [comment]
Also, should ensure that the serialized coordinates are on screen.

If someone using resolution `1920x1080` has the serialized UI values at `1000, 700`, and then switches to `800x640`, ensure the UI is still on screen.

**Comment by Simpy** [comment]
Just woke up and am about to start game, so if you need any testing let me know, if you are on discord I have that window open while playing...

**Comment by Therzok** [comment]
Yup, I found some APIs that might help with this, I'm going to give them a try now:
https://docs.unity3d.com/ScriptReference/RectTransformUtility.PixelAdjustPoint.html

**Comment by Simpy** [comment]
Strange, I tried searching for you yesterday before leaving the message here, but you never showed up. I'm NZ_Wanderer on discord.
